src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/SetAzurePolicyDefinition.cs

```diff
@@ -94,6 +94,12 @@ public class SetAzurePolicyDefinitionCmdlet : PolicyCmdletBase
         [ValidateNotNullOrEmpty]
         public Guid? SubscriptionId { get; set; }
 
+        /// <summary>
+        /// Gets or sets the policy definition input object parameter.
+        /// </summary>
+        [Parameter(ParameterSetName = PolicyCmdletBase.InputObjectParameterSet, Mandatory = true, ValueFromPipeline = true, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.SetPolicyDefinitionInputObjectHelp)]
```

Comment:
@mentat9 for the `-InputObject` parameters you are adding, you can remove the `ValueFromPipelineByPropertyName` attribute (you only need `ValueFromPipeline`)